### PR TITLE
use dot only if inbetween things

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump, tiny pr

File: shared/fs/common/tlf-info-line.tsx [hunk 1]
```diff
@@ -87,18 +80,34 @@ const getText = (props: Props) => {
...
+  const text = getText(props)
+  return (
+    <Kb.Box2
+      direction="horizontal"
+      fullWidth={true}
+      centerChildren={props.mode === 'default'}
+      alignItems="center"
+    >
+      {prefix}
+      {prefix && (reset || text) ? dot : null}
```
Comment (buoyad) [inline shared/fs/common/tlf-info-line.tsx]: I still see some of these
image

Comment (buoyad) [reply shared/fs/common/tlf-info-line.tsx]: LMK if I can help debug

Comment (chrisnojima) [reply shared/fs/common/tlf-info-line.tsx]: Yeah cause I was never able to repro in the first place

Comment (songgao) [reply shared/fs/common/tlf-info-line.tsx]: to reproduce maybe just set this to 0? https://github.com/keybase/client/blob/938410b53d87b0d651718e9e63f4936e09b1d3c4/shared/actions/fs/index.tsx#L108

File: shared/fs/common/tlf-info-line.tsx [hunk 2]
```diff
@@ -87,18 +80,34 @@ const getText = (props: Props) => {
...
-)
+const TlfInfoLine = (props: Props) => {
+  const prefix = <PrefixText {...props} />
+  const dot = (
+    <Kb.Text
+      type="BodySmall"
+      style={props.mode === 'default' ? styles.textDefault : styles.textRow}
+      lineClamp={props.mode === 'row' && Styles.isMobile ? 1 : undefined}
+    >
+      •&nbsp;
```
Comment (songgao) [inline shared/fs/common/tlf-info-line.tsx]: Does `&nbsp;` work on mobile?

File: shared/fs/common/tlf-info-line.tsx (hunk 1 above)
Comment (chrisnojima) [reply shared/fs/common/tlf-info-line.tsx]: ok so the issue is the siblings i'm checking always exist, even though inside they render to 0. they don't really serve a purpose being actual components so i just made them functions so we get either null or some other component and this logic works

**Pull request merged**